app/controllers/SchemeDetailsController.scala

```diff
@@ -71,7 +71,7 @@ class SchemeDetailsController @Inject()(appConfig: FrontendAppConfig,
                   administrators(request.psaId.id, scheme),
                   srn.id,
                   isSchemeOpen,
-                  false
+                  true
```

Comment:
Better to name boolean parameters, i.e. isSchemeLocked = false

Reply:
fixed the flag name